eZ/Publish/Core/FieldType/TextLine/Type.php

```diff
@@ -64,9 +64,9 @@ public function validateValidatorConfiguration($validatorConfiguration)
                 switch ($name) {
                     case 'minStringLength':
                     case 'maxStringLength':
-                        if ($value !== null && !is_integer($value)) {
+                        if ($value !== null && (!is_integer($value) || $value < 1)) {
```

Comment:
updated and moved the validation to StringLengthValidator instead?... will that be correct?